koopmans/workflows/_koopmans_dscf.py

```diff
@@ -385,6 +385,11 @@ def perform_initialisation(self) -> None:
             if abs(Efin - Eini) > 1e-6 * abs(Efin):
                 raise ValueError(f'Too much difference between the initial and final CP energies: {Eini} {Efin}')
 
+            # Add to the outdir of dft_init a link to the files containing the Wannier functions
+            dst = Path(f'{calc.parameters.outdir}/{calc.parameters.prefix}_{calc.parameters.ndw}.save/K00001/')
+            for file in ['evc_occupied1.dat', 'evc_occupied2.dat', 'evc0_empty1.dat', 'evc0_empty2.dat']:
+                utils.system_call(f'ln -sf {restart_dir}/{file} {dst}')
```

Comment:
Please replace with `utils.symlink` so that we don't rely on `ln`

Reply:
I did not think of a better way to make `symlink` to emulate the `ln -sf` command.

Reply:
Can we not just add an additional argument `overwrite` to `utils.symlink`? And then prior to 
```
        if exist_ok:
            try:
                dest.symlink_to(src)
            except FileExistsError:
                pass
        else:
            dest.symlink_to(src)
```
simply add something like (N.B. this is untested)
```
        if overwrite:
            # The equivalent of ln -f
            dest.unlink(missing_ok=True)
```

Reply:
Sorry I think the lines showed above were outdated. I had done something similar to what you suggested. I just modified it to use this `.unlink` function rather than resorting to `os.remove`.